src/App.tsx

```diff
@@ -15,36 +16,124 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+export enum SortType {
+  SORT_BY_ALPHABET = 'Sort alphabetically',
+  SORT_BY_LENGTH = 'Sort by length',
+}
+
+export interface Change {
+  sortBy: SortType | null;
+  isReversed: boolean;
+}
+
+const sortByAlphabet = (isReversed: boolean, good1: string, good2: string) =>
+  isReversed ? good2.localeCompare(good1) : good1.localeCompare(good2);
+
+function getSortedGoods(goods: string[], { sortBy, isReversed }: Change) {
```

Comment:
The function's name `getSortedGoods` should be renamed to more clearly represent its purpose, such as `sortGoods` or `getGoodsSorted`, to adhere to the code style guideline of starting function names with a verb. [CODE STYLE]